### PR TITLE
style: resolve eslint warnings

### PR DESCRIPTION
**Pull request by ChristianMurphy**
define `static get observeredAttributes` only once.
update and re-render component when attribute changed callback runs.
extract urlMessages function.
use async await for network calls.

### CODE REVIEW

File: src/my-card.js [hunk 1]
```diff
@@ -2,141 +2,141 @@
...
+    return ["cssPath", "id", "messagesPath"];
   }
 
   constructor(...args) {
-    super(...args);
-    var idComponent, urlMessages, urlCss;
-    let path = `${this.getAttribute("messagesPath")}`;
-    idComponent = `${this.getAttribute("id")}`;
-    urlCss = `${this.getAttribute("cssPath")}`;
+    const self = super(...args);
```
Comment (ChristianMurphy) [inline src/my-card.js]: Use `self` to allow additional polyfills to optionally be used.
See: https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements#Polyfills_versus_classes

File: src/my-card.js [hunk 2]
```diff
@@ -2,141 +2,141 @@
...
+   * Build the url of the messages according to host page language
+   * Supported language: en-US, fr-FR, es-ES, nl-NL (Can be extended...)
+   * if host language is unknowed default to en-US messages
+   */
+  urlMessages(path, pageLang) {
+    switch (pageLang) {
+      case "fr-FR":
+      case "es-ES":
+      case "en-US":
+      case "nl-NL": {
```
Comment (ChristianMurphy) [inline src/my-card.js]: Since page lang is included in the template, case fallthrough can be used to simplify this code.

File: src/my-card.js [hunk 3]
```diff
@@ -2,141 +2,141 @@
...
+    const articleList = shadowroot.querySelector(".card__social");
+    const articleToggle = shadowroot.querySelector(".card__share");
     articleList.classList.toggle("card__social--active");
     articleToggle.classList.toggle("share-expanded");
 
-    if (articleToggle.getAttribute("aria-expanded") === "false") {
-      articleToggle.setAttribute("aria-expanded", "true");
-    } else {
-      articleToggle.setAttribute("aria-expanded", "false");
+    articleToggle.toggleAttribute("aria-expanded");
```
Comment (ChristianMurphy) [inline src/my-card.js]: Similar to classList, attributes can be toggled.
https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute

File: src/my-card.js (hunk 1 above)
Comment (cousquer) [reply src/my-card.js]: 👍

File: src/my-card.js (hunk 2 above)
Comment (cousquer) [reply src/my-card.js]: yes

File: src/my-card.js (hunk 3 above)
Comment (cousquer) [reply src/my-card.js]: yes it should

**Pull request merged**